HW_1/task_3.py

```diff
@@ -0,0 +1,44 @@
+"""
+Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости,
+в которой находится эта точка (или на какой оси она находится).
+
+Ввод: два значения типа <int>
+Вывод: значение типа <int> либо значение типа <str>
+
+Пример:
+
+34
+-30
+4
+
+2
+4
+1
+
+-34
+0
+Точка на отрицательной части оси абсцисс
+"""
+
+x = int(input("Введите координату точки x: "))
+y = int(input("Введите координату точки y: "))
+
+if x > 0 and y > 0:
```

Comment:
Если в условии присутствует повторяющееся значение, операторы сравнения или равенства, а также оператор `and`, то это условие можно свернуть относительно повторяющегося значения.
```suggestion
if x > 0 < y:
```
Остальные условия попробуйте свернуть сами.

Reply:
не понял как свернуть повторяющиеся значения ... то есть вместо and сразу пишем знак больше или меньше (от и до), так?

Reply:
> не понял как свернуть повторяющиеся значения ... то есть вместо and сразу пишем знак больше или меньше (от и до), так?

Я ведь в Suggested change указал как. Нужно сделать по аналогии.